tests/common/include/tests/common/tools/create_toy_geometry.hpp

```diff
@@ -908,17 +971,19 @@ auto create_toy_geometry(vecmem::memory_resource &resource,
         std::vector<scalar> m_half_x_min_y = {8.4, 8.4};
         std::vector<scalar> m_half_x_max_y = {12.4, 12.4};
         std::vector<scalar> m_tilt = {0., 0.};
+        material<scalar> mat = silicon_tml<scalar>();
+        scalar thickness = 0.15 * unit_constants::mm;
```

Comment:
Just a quick question: These correspond to the parameters of the acts generic detector?

Reply:
Actually I am not 100% sure. Can you check the following code if it is right value?

https://github.com/acts-project/acts/blob/4adaf4c6125046ae2d8d835a17e976e810e86c62/Examples/Detectors/GenericDetector/include/ActsExamples/GenericDetector/BuildGenericDetector.hpp#L213

Reply:
Hm, looks like it is correct for the modules... I don't know much about the material though. The value seems reasonable anyway, so let's put it in like that